examples/gitlab-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,63 @@
+---
+specversion: 0.0.2
+id: gitlab-atomicapp
+
+metadata:
+  name: Gitlab App
+  appversion: 1.2.0
+  description: Gitlab.
+graph:
+  - name: redis
```

Comment:
Nulecule supports using graphs as well as defining multiple components. I agree on the part that using graphs is recommended. IMO, we should have the example set showing different ways supported by Nulecule. This example showcase using different tiers by enlisting the respective artifacts and not using any graphs.